### PR TITLE
PR: Wishlist

### PR DESCRIPTION
**Pull request by pavan6115**
- Implemented WishList Functionality :
  - Add to Wishlist
  - Remove from Wishlist
  - Clear Wishlist
  - Move from WishList to Cart 

https://user-images.githubusercontent.com/55844228/187024512-724ce33e-89bf-476f-8eac-18536f5990af.mov

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-makeServer();
+makeServer()
 
 ReactDOM.render(
   <React.StrictMode>
-      <CartProvider>
+    <CartProvider>
+      <WishListProvider>
         <DataProvider>
           <Router>
```
Comment (ashishkatiyar2404) [inline src/index.js:19]: Router should be on top of all Providers

File: src/pages/Wishlist/WishList.css [hunk 2]
```diff
@@ -1,86 +1,107 @@
...
 .redirect__cart__btn {
-    font-size: 1.5rem;
-    margin: 1.5rem 0rem;
+  font-size: 1.5rem;
+  margin: 1.5rem 0rem;
 }
 
 .redirect__cart__btn:hover {
-    background-color: rgba(242, 92, 5, 0.774);
+  background-color: rgba(242, 92, 5, 0.774);
```
Comment (ashishkatiyar2404) [inline src/pages/Wishlist/WishList.css:78]: use color variables

**Review by ashishkatiyar2404 — APPROVED** [review]
Suggested some changes. 
your code functionality looks good to me , you can merge and work on other functionality

File: src/index.js (hunk 1 above)
Comment (pavan6115) [reply src/index.js:19]: Will do, thanks 👍

File: src/pages/Wishlist/WishList.css (hunk 2 above)
Comment (pavan6115) [reply src/pages/Wishlist/WishList.css:78]: ok

**Review by ayush65 — APPROVED** [review]
Code looks clean good to go

**Pull request merged**